### PR TITLE
Half-fixes

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

**Comment by Drayano** [comment]
This PR broke Persona 5 title screen
rpcs3_2018-01-30_21-57-12

log file 
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/1679356/RPCS3.log.gz)

**Comment by Zangetsu38** [comment]
@Drayano clean youir LLVM cache en retry

**Comment by Drayano** [comment]
@Zangetsu38 I already did that

**Comment by Lemiru** [comment]
This PR fixes Zone of the enders HD collection not working with patch installed

**Comment by Nekotekina** [comment]
@Drayano @Lemiru Thanks for testing. Is it PPU or SPU that cause this bug? You can check by choosing interpreters in settings.

**Comment by AniLeo** [comment]
I can't reproduce that Persona 5 bug, works fine here with both Recompilers

**Comment by Drayano** [comment]
I tried out every possible combination of PPU and SPU options and the problem doesn't seem to go away, i should precise that i'm on Intel UHD 630 igpu by the way.

**Comment by isshininu** [comment]
This PR cause a glitch during shaders compiling in Demon's Souls - everything is white with distant lights visible through the fog.

**Comment by AniLeo** [comment]
Regarding the P5 bug, it's likely to be caused by #4127, it's happening on master

image

File: .gitmodules [hunk 1]
```diff
@@ -7,8 +7,8 @@
 	ignore = dirty
 [submodule "llvm"]
 	path = llvm
-	url = https://github.com/llvm-mirror/llvm
-	branch = release_40
+	url = https://github.com/RPCS3/llvm
+	branch = release_60
```
Comment (Zangetsu38) [inline .gitmodules]: you leave that ?

Comment (AniLeo) [reply .gitmodules]: It's required, look at the fork

Comment (Zangetsu38) [reply .gitmodules]: No required for moment, submdule is not updated with this URL

Comment (AniLeo) [reply .gitmodules]: It shouldn't be building right now ¯\_(ツ)_/¯but it will be required to use the fork

Comment (Zangetsu38) [reply .gitmodules]: you don't understund, for moment neko don't have updated submodule with this url, using old llvm4, so why changed that ? This just broken git for clone submodule

Comment (Nekotekina) [reply .gitmodules]: What happens at clone submodule? I thought it still would be able to fetch the 4.0 commit.

Comment (Zangetsu38) [reply .gitmodules]: hmm after Verification Test with try update submodule is good.
Using "Git submodule sync" for update url with folder already exist, and "git submodule update --init" for update works

**Pull request merged**